### ISSUE TITLE
Playlists: is there a simpler way to manage them?

### ISSUE THREAD

**Issue by RapFan89** [DanielnetoDotCom/YouPHPTube#1284]
Hi there,
as i said in the title, is  possible to have a multiselection tool in which i can select all my videos and create a new playlist based on them?

**Comment by DanielnetoDotCom** [comment]
For now you must add one by one video on the playlist

**Comment by RapFan89** [comment]
We have around 300+ videos,  this is not going to be a cakewalk.

**Comment by birdie60** [comment]
Another idea is to use phpMyAdmin and do it in the database. You need to know SQL, then this works fine.

**Comment by RapFan89** [comment]
> 
> 
> Another idea is to use phpMyAdmin and do it in the database. You need to know SQL, then this works fine.

Thanks for the advice.

We have all videos under categories, we have in not-in-list mode all the videos, we created playlists that have the same name of the categories, and through sql we've associated all the videos that belong to a category to the corresponding playlist. Now we want to share the playlist, is it possibile?
The idea is to hide the videos, and show to the user that has the link only a group per time.
